### DISCUSSION TITLE
I accidentally remove myself as a sole owner of my organization. how to restore?

### DISCUSSION THREAD

**Discussion by BayaraaBasan** [community/community#68499] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I accidentally remove myself as a sole owner of my organization. Then i created support ticket on **support.github.com**. Anyone didn't contact me. What can i do now?

**Comment by Jonak-Adipta-Kalita** [comment, marked as answer]
You will have to wait, I suppose, from next time, keep 2 admins for, this kind of stuff might happen again

**Reply by Jonak-Adipta-Kalita** [reply]
I used so many commas... wut....

**Reply by mehraUTD** [reply]
what is the solution? I have so many project there

**Reply by mehraUTD** [reply]
This is my organization : https://github.com/UTDClassroomOrg and i was the only owner and somehow accidentally I removed myself !
